### ISSUE TITLE
add non-plural option kubectl config get-context (no trailing s)

### ISSUE THREAD

**Issue by mlda065** [kubernetes/kubectl#508]
tldr: Add `kubectl config get-context` as a synonym for `kubectl config get-contexts`

**Is this a request for help?**: No

**What keywords did you search in Kubernetes issues before filing this one?** (If you have found any duplicates, you should instead reply there.): 

* `get-context`

---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):

Feature Request

**Kubernetes version** (use `kubectl version`):

```
Client Version: version.Info{Major:"1", Minor:"11", GitVersion:"v1.11.0", GitCommit:"91e7b4fd31fcd3d5f436da26c980becec37ceefe", GitTreeState:"clean", BuildDate:"2018-06-27T20:17:28Z", GoVersion:"go1.10.2", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"9", GitVersion:"v1.9.6", GitCommit:"9f8ebd171479bec0ada837d7ee641dec2f8c6dd1", GitTreeState:"clean", BuildDate:"2018-03-21T15:13:31Z", GoVersion:"go1.9.3", Compiler:"gc", Platform:"linux/amd64"}
```

* If you want info about pods, you say 
   - `kubectl get pods` or
   - `kubectl get pod`
* If you want info about services, you say 
   - `kubectl get services` or
   - `kubectl get service`
* If you want info about deployments, you say 
   - `kubectl get deployments` or
   - `kubectl get deployment`
* If you want info about contexts, you say `kubectl config get-contexts`, but you *cannot* say `kubectl config get-context`.

May I suggest making `get-contexts` plural-agnostic, like everything else?

I'd also like to suggest changing the syntax entirely from `kubectl config get-context` to `kubectl get contexts`, but that's probably a more audacious change.
